### PR TITLE
Fix autoplay on error

### PR DESCRIPTION
**Pull request by adgad**
/cc @AlbertoElias 
I think this fixes the issue, and also will play the video on clicking the placeholder (if no ad)

### CODE REVIEW

File: src/js/video.js [hunk 1]
```diff
@@ -199,6 +199,7 @@ class Video {
 			this.videoEl.style.display = 'block';
 			if (!this.opts.advertising) {
 				this.videoEl.load();
+				this.videoEl.play();
```
Comment (AlbertoElias) [inline src/js/video.js:201]: it should work without the play, load just plays the video from the beginning if I understood correctly

File: src/js/ads.js [hunk 2]
```diff
@@ -218,7 +218,9 @@ class VideoAds {
 
 	adErrorHandler() {
 		this.adsManager && this.adsManager.destroy();
-		this.video.videoEl.play();
+		this.video.containerEl.removeChild(this.adContainerEl);
+		this.video.placeholderEl && this.video.placeholderEl.removeEventListener('click', this.playAdEventHandler);
```
Comment (AlbertoElias) [inline src/js/ads.js:226]: shouldn't it remove the other event handler for the overlay as well?

File: src/js/ads.js [hunk 3]
```diff
@@ -218,7 +218,9 @@ class VideoAds {
 
 	adErrorHandler() {
 		this.adsManager && this.adsManager.destroy();
-		this.video.videoEl.play();
+		this.video.containerEl.removeChild(this.adContainerEl);
```
Comment (AlbertoElias) [inline src/js/ads.js:221]: also remove the overlayEl if there is one

File: src/js/video.js (hunk 1 above)
Comment (adgad) [inline src/js/video.js:201]: It didn't seem to play for me without .play()

Comment (AlbertoElias) [inline src/js/video.js:201]: then I'd remove the load

**Comment by adgad** [comment]
All look good now?

**Comment by AlbertoElias** [comment]
yup! Thanks! :D

**Pull request merged**